src/com/sap/piper/Utils.groovy

```diff
@@ -149,7 +149,7 @@ static String downloadSettingsFromUrl(script, String url, String targetFile = 's
     return targetFile
 }
 
-/*
+/**
```

Comment:
Unrelated. Maybe it is intentionally a normal java command rather than a javadoc command. Anyway we don't generate javadoc. Would like to suggest to keep it as it is.

Reply:
Will revert...